src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -134,6 +134,10 @@
     "DESCRIPTION": "Apply stsjorbsmod:Slow to ALL enemies. NL Add !M! stsjorbsmod:Web into your hand.",
     "UPGRADE_DESCRIPTION": "Apply stsjorbsmod:Slow to ALL enemies. NL Add !M! stsjorbsmod:Webs into your hand."
   },
+  "stsjorbsmod:Expose": {
+    "NAME": "Expose",
+    "DESCRIPTION": "Deal !D! damage. NL If there are enemies still alive, lose !M! HP."
```

Comment:
Since jorbs clarified that we *do* want to suppress the HP loss if there are only minions still alive, maybe we can reword this to be more clear? `If there are still enemies active`? `If any enemies are still willing to fight`? `If any enemies are still able to fight`?